kubernetes/ansible/roles/flink-jobs-deploy/defaults/main.yml

```diff
@@ -70,3 +70,8 @@ cloud_storage_key: ""
 cloud_storage_secret: ""
 cloud_storage_container: ""
 cloud_storage_endpoint: ""
+
+cloudstorage_replace_absolute_path: true
+cloudstorage_base_path: "https://sunbirdstagingpublic.blob.core.windows.net"
+cloudstorage_metadata_list: '["appIcon","posterImage","artifactUrl","downloadUrl","variants","previewUrl","pdfUrl"]'
+valid_cloudstorage_base_urls: '["https://sunbirdstagingpublic.blob.core.windows.net"]'
```

Comment:
@krgauraw instead of hardcoding can we take these values from env

Reply:
@pallakartheekreddy these are default variables. anyone can override from private devops repo. So no changes are required here.